fpga/src/main/scala/arty/Platform.scala

```diff
@@ -51,6 +51,7 @@ class E300ArtyDevKitPlatformIO(implicit val p: Parameters) extends Bundle {
 //-------------------------------------------------------------------------
 
 class E300ArtyDevKitPlatform(implicit val p: Parameters) extends Module {
+  //val sys = Module(LazyModule(new E300ArtyDevKitSystem).module) This can be DigitalTop?
```

Comment:
Is this just remnants from some other testing?

Reply:
Oh. Yea. This should be deleted.